src/control/cmd/dmg/system.go

```diff
@@ -134,6 +135,20 @@ func (cmd *systemQueryCmd) Execute(_ []string) (errOut error) {
 	return resp.Errors()
 }
 
+type systemEraseCmd struct {
+	logCmd
+	ctlInvokerCmd
+}
+
+func (cmd *systemEraseCmd) Execute(_ []string) error {
+	resp, err := control.SystemErase(context.Background(), cmd.ctlInvoker, new(control.SystemEraseReq))
+	if err != nil {
+		return err
+	}
+
```

Comment:
so no json handling here? is the rationale that only the error is relevant (and the json error presentation is implicit)?

Reply:
Right. From my perspective, either the system erase operation succeeded (which means that the system is erased and there's no detailed information to convey), or it failed and the error will contain more information about the details.